packages/data/addon/mirage/handlers/graphql.js

```diff
@@ -477,7 +477,6 @@ const OPTIONS = {
       if (op === 'UPSERT' && existingQueries.length === 0) {
         const node = asyncQueries.insert({
           id: data.id,
-          asyncAfterSeconds: 10,
```

Comment:
why is this being removed? Should it just be changed to `asyncAfterSeconds: data.asyncAfterSeconds,`?

Reply:
This was a mistake. I have added it back. thank you